engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscriptions.routes.js

```diff
@@ -60,7 +60,7 @@ angular.module('Bastion.subscriptions').config(['$stateProvider', function ($sta
         }
     })
     .state('subscription.content-hosts', {
-        url: '/content-hosts',
+        url: '/content-hosts?search',
```

Comment:
same question here, didn't think this was needed

Reply:
its needed for the search parameter here https://git.io/v5gNZ

Reply:
The search query string param is needed any time you use state to transition with a search as opposed to a page load (i.e. `ui-sref` or `$state.go()`).